README.md

```diff
@@ -246,11 +244,11 @@ There are several ways to facilitate unit testing when using Batcher. Detailed d
 
 ## Rate Limiting
 
-The AzureSharedResource rate limiter works like this...
+The SharedResource rate limiter works like this...
 
 ![lease](./docs/images/lease.png)
```

Comment:
This img is out of date

Reply:
We did update the images, maybe they are cached. Are you seeing the one that says LeaseManager at the right?

Reply:
Very strange. I refreshed before and it showed the old versions, but now it shows the updated images. Must have been caching, but maybe server-side? Anyhow, I'll mark the image comments as resolved.